Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -145,6 +145,14 @@ internal struct PaymentsResponse: Response {
         case amount
     }
     
+    var isSuccessful: Bool {
```

Comment:
```suggestion
    var nonError: Bool {
```

Since `pending` is not yet a "success"

😁